recipes/efa_install.rb

```diff
@@ -57,3 +57,15 @@
   EFAINSTALL
   not_if { ::Dir.exist?('/opt/amazon/efa') && !efa_gdr_enabled?}
 end
+
+# EFA installer v1.11.0 removes libibverbs-core, which contains hwloc-devel during install
+# hwloc-devel is needed to compile SGE
+# Reinstall hwloc-devel by itself explicitly
+package "install hwloc-devel explicitly" do
+  package_name value_for_platform(
+    'default' => "hwloc-devel",
+    'ubuntu' => { 'default' => "libhwloc-dev" }
```

Comment:
should we also install hwloc-devel/libhwloc-dev on ubuntu and centos platform although there is no failure?

Reply:
Problem currently only occurs on AL2, but EFA mentioned that other OSs might start to change to this naming convention that is causing the trouble on AL2.
Proposing to do this for all OSs, just to be safe.

Reply:
Nice!